Library/Homebrew/formula_auditor.rb

```diff
@@ -199,14 +199,28 @@ def audit_name
       "LGPL-3.0" => ["LGPL-3.0-only", "LGPL-3.0-or-later"],
     }.freeze
 
+    INCOMPATIBLE_LICENSES = %w[
+      JSON
+    ].freeze
+
+    INCOMPATIBLE_LICENSE_PREFIXES = %w[
+      BUSL
+      CC-BY-NC
+      Elastic
+      SSPL
+    ].freeze
```

Comment:
Might warrant comment(s) explaining why these are incompatible.

Reply:
I added some simple comments/refs. In future, may need to move to a JSON file, which can be used to better document reason.